include/operations/blas2_trees.h

```diff
@@ -116,9 +116,9 @@ struct SumMatrixColumns {
 
   bool valid_thread(cl::sycl::nd_item<1> ndItem) const;
 
-  value_t eval(index_t i);
+  value_t eval(index_t i) const;
```

Comment:
I think we can drop the `const` here to make this `eval` method conformant with the `eval` functions of other Level 2 operators.